fastlane/lanes/localization.rb

```diff
@@ -223,50 +266,80 @@
+    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'jetpack_metadata')
 
-    # @TODO: Make the `fastlane/jetpack_metadata/en-US/release_notes.txt` path be the source of truth for the original copies in the future.
+    # FIXME: We should make the `fastlane/jetpack_metadata/default/release_notes.txt` path be the source of truth for the original copies in the future.
     # (will require changes in the `update_appstore_strings` lane, the Release Scenario, the MC tool to generate the announcement post…)
     #
-    # In the meantime, since GlotPress doesn't have the `en-US` notes because those are the ones used as originals, just copy the file to the right place for `deliver` to find
-    metadata_directory = File.join(PROJECT_ROOT_FOLDER, 'fastlane', 'jetpack_metadata')
-    release_notes_source = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
-    FileUtils.cp(release_notes_source, File.join(metadata_directory, 'en-US', 'release_notes.txt'))
+    # In the meantime, just copy the file to the right place for `deliver` to find, for the `default` pseudo-locale which is used as fallback
+    release_notes_source = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt')
+    FileUtils.cp(release_notes_source, File.join(metadata_directory, 'default', 'release_notes.txt'))
+
+    # Download metadata translations from GlotPress
+    download_localized_app_store_metadata(
+      glotpress_project_url: GLOTPRESS_JETPACK_METADATA_PROJECT_URL,
+      locales: JETPACK_METADATA_GLOTPRESS_LOCALE_CODES,
+      metadata_directory: metadata_directory,
+      commit_message: 'Update Jetpack metadata translations'
+    )
+  end
+
+  def download_localized_app_store_metadata(glotpress_project_url:, locales:, metadata_directory:, commit_message:)
+    # FIXME: Replace this with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
 
-    metadata_path = File.join(metadata_directory, '**', '*.txt')
-    git_add(path: metadata_path, shell_escape: true)
+    locales_map = GLOTPRESS_TO_ASC_METADATA_LOCALE_CODES.slice(*locales)    
+    what_new_key = "v#{ios_get_app_version}-whats-new"
+    gp_downloadmetadata(
+      project_url: glotpress_project_url,
+      target_files: APPSTORECONNECT_METADATA_KEYS_TO_FILES_MAP,
+      locales: locales_map,
+      download_path: metadata_directory
+    )
+    files_to_commit = [File.join(metadata_directory, '**', '*')]
+
+    # Ensure that none of the `.txt` files in `en-US` would accidentally override our originals in `default` by mistake
+    APPSTORECONNECT_METADATA_KEYS_TO_FILES_MAP.values.map { |h| h[:desc] }.each do |file|
+      en_file_path = File.join(metadata_directory, 'en-US', file)
+      UI.user_error!("File `#{en_file_path}` would override the same one in `#{metadata_directory}/default`, but `default/` is our source of truth for those. " \
```

Comment:
Minor suggestion that, IMHO, makes the comment and error slightly more concise.

```suggestion
    # Ensure that none of the `.txt` files in `en-US` would accidentally override our originals in `default`
    APPSTORECONNECT_METADATA_KEYS_TO_FILES_MAP.values.map { |h| h[:desc] }.each do |file|
      en_file_path = File.join(metadata_directory, 'en-US', file)
      UI.user_error!("File `#{en_file_path}` would override the same one in `#{metadata_directory}/default`, but `default/` is the source of truth. " \
```